resources/sdk/purecloudgo/templates/apiclient.mustache

```diff
@@ -96,6 +96,46 @@ func getPathValue(pc *ProxyConfiguration, key string) (string, bool) {
 	return "", false
 }
 
+func getConfUrl(c *Configuration, path string, extendedPath string, overrideHost string) *url.URL {
+	var uri *url.URL
+	gateWayConfiguration := c.GateWayConfiguration
+
+	uri, _ = url.Parse(fmt.Sprintf("https://api.%s%s", c.BasePath, extendedPath))
```

Comment:
if we are not setting url explicitly previously, dont think we need to change that behaviour. same applies for login as well. as far as i see go sdk doesnt explcitly differentiate api and login url , since it has a single callapi method with "path" getting injected to it.  dont think we have to change that behaviour. just check for gateway params and override them if present.